### DISCUSSION TITLE
What is CSS?

### DISCUSSION THREAD

**Discussion by xlgmokha** [CodeChica/plus-plus#31] [General]
Can you help me understand when and where I would want to use CSS?

**Comment by daitanfutekina** [comment]
the code that will make it have a unique or good looking appearance, it can change the colors, font, ete, its the art class of coding

**Comment by superrrk** [comment]
You would use CSS **when** you need to style the content you outlined in your HTML code. This way, your content is organized, positioned, colored, etc., the way you would like it to. You would **want** to use CSS when adding your own style and personal creative touches to your HTML content to make it user-friendly and pleasing to the eye.

**Comment by IxchelO** [comment]
CSS is code you use in your HTML code when trying to make your product decorated or pleasing. You use it when doing fonts, colors, pictures, etc.

**Comment by code2417** [comment]
CSS is a styling language often used with HTML to make the webpage user-friendly and pretty. CSS can add elements like fonts, colors, images, textboxes, etc.
